docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -938,11 +931,36 @@ research may be attempting to elucidate and alleviate. Race-conscious research,
 focus from race to racism as a key determinant of illness and health. For examples of outdated race-based practices in medicine, 
 and their race-conscious alternatives, see Table 1 from Cerdeña et al.’s 2020 paper [Cerdeña_2020]_. 
 
+
+Reporting results accurately
+----------------------------
+
+We often need to be very concise, and can’t surface all the complexity of race and ethnicity every time we reference them. 
+So, we take the following approach:
+
+Somewhere in the paper/report, we explain in a fair amount of detail the race and ethnicity categorizations we used, why we used them that way, etc. 
```

Comment:
Before the etc maybe add: the race and ethnicity groups used in input data, how we completed our analysis across different datasets, etc.